transport_nantes/topicblog/test_tn_links.py

```diff
@@ -100,8 +100,8 @@ def test_external_url_button(self):
     def test_internal_url(self):
         the_slug = 'my-slug'
         the_label = 'my-label-text'
-        markdown = f'[[slug:{the_slug}]](({the_label}))'
-        html = slug.tbi_slug(the_slug, the_label)
+        markdown = f'[[slug:{the_label}]](({the_slug}))'
```

Comment:
Then you put the label next to the slug, it's counterintuitive. 
I do agree that it makes more sense to have the [[label]]((slug)) because it fits the usual markdown pattern, but I think you should also update the parser to not use "slug" arg for label 
It's line 229 in tn_links.py

Reply:
The pattern (and the "don" directives are an unfortunate exception) is `[[verb:user-text]]((direct-object))`, for some very permissive definition of "verb" and "direct-object".

Are you suggesting a different user interface than `[[slug:text]]((item_slug))`?  What do you think the right thing to do is?

Reply:
BTW, I'm perfectly fine with the idea that we should migrate our markdown extensions to a more logical schema.  It's not yet hard to do.

If we want to go that way, we should make a design doc.  But we can start that conversation here.

Reply:
Was thinking about the code, mostly, because reading it and seeing "slug:" I'd expect to have the slug argument right next to it, and it's now the label. 
but I see what you mean in "verb" now that you say it, maybe it's just the "slug" verb that is just not very clear from a user (user as in someone who writes articles for mobilitains) standpoint. 
`[[bouton_vers_article: {the_label}]](({the_slug}))`
would make more sense, maybe ? 
Though it wasn't the objective of this PR initially :grimacing:

Reply:
As I said, I'm open to a global rethink and design doc. ;-)

What do you think on this PR in particular?

Reply:
No fixed idea on the design doc, just that if we think the vocabulary or syntax needs modifying, better now than a year from now.

So do you lgtm?

Reply:
PR: LGTM 

Agreed with the need of design doc ! Will you open the issue  ?

Reply:
Done.